### ISSUE TITLE
Ablation experiment

### ISSUE THREAD

**Issue by Yffy123456** [MASILab/3DUX-Net#14]
Thank you for your contribution to the paper and code. I would like to ask you about the ablation experiment. The code should take deep convolution (1) how does it become standard convolution (2) kernel=7*7*7 self.dwconv = nn.Conv3d(dim, dim, kernel_size=7, padding=3groups=dim) if the kernel changes to 7,13, etc., does it change kernel_size

**Comment by leeh43** [comment]
Hi, thank you for your interest to our work! For ablation studies, the standard convolution is corresponding to the normal convolution, instead of depthwise convolution. We implement the normal convolution like nn.Conv3d(in_channels=48, out_channels=48, kernel_size, stride, padding, group=1), instead of group=48.

For your second question, I am a little bit confused, but it is definitely changing the kernel_size, if the kernels parameter changes from 7, 9,  11, 13 in our ablation studies. Also, we need to change the padding parameter, because we want to ensure the output feature dimension is the same as the input feature dimension with depthwise convolution.
